Intersect.Client/Interface/Game/EscapeMenu.cs

```diff
@@ -36,55 +21,39 @@ public EscapeMenu(Canvas gameCanvas) : base(gameCanvas, "EscapeMenu")
         Width = gameCanvas.Width;
         Height = gameCanvas.Height;
 
-        mContainer = new ImagePanel(this, "EscapeMenu");
+        // Create the container
+        var container = new ImagePanel(this, "EscapeMenu");
 
-        mTitle = new Label(mContainer, "TitleLabel")
-        {
-            Text = Strings.EscapeMenu.Title,
-        };
+        // Title Label
+        _ = new Label(container, "TitleLabel") { Text = Strings.EscapeMenu.Title };
 
-        mSettingsWindow = new SettingsWindow(gameCanvas, null, this);
+        // Settings Window and Button
+        _settingsWindow = new SettingsWindow(gameCanvas, null, this);
 
-        mSettings = new Button(mContainer, "SettingsButton")
-        {
-            Text = Strings.EscapeMenu.Settings
-        };
+        var buttonSettings = new Button(container, "SettingsButton") { Text = Strings.EscapeMenu.Settings };
+        buttonSettings.Clicked += (s, e) => OpenSettingsWindow(true);
 
-        mSettings.Clicked += Settings_Clicked;
+        // Character Select Button
+        _buttonCharacterSelect = new Button(container, "CharacterSelectButton") { Text = Strings.EscapeMenu.CharacterSelect };
+        _buttonCharacterSelect.Clicked += _buttonCharacterSelect_Clicked;
 
-        mGoToCharacterSelect = new Button(mContainer, "CharacterSelectButton")
-        {
-            Text = Strings.EscapeMenu.CharacterSelect
-        };
+        // Logout Button
+        var buttonLogout = new Button(container, "LogoutButton") { Text = Strings.EscapeMenu.Logout };
+        buttonLogout.Clicked += buttonLogout_Clicked;
 
-        mGoToCharacterSelect.Clicked += GoToCharacterSelect_Clicked;
+        // Exit to Desktop Button
+        var buttonQuit = new Button(container, "ExitToDesktopButton") { Text = Strings.EscapeMenu.ExitToDesktop };
+        buttonQuit.Clicked += buttonQuit_Clicked;
 
-        mLogout = new Button(mContainer, "LogoutButton")
-        {
-            Text = Strings.EscapeMenu.Logout
-        };
-
-        mLogout.Clicked += Logout_Clicked;
-
-        mExitToDesktop = new Button(mContainer, "ExitToDesktopButton")
-        {
-            Text = Strings.EscapeMenu.ExitToDesktop
-        };
-
-        mExitToDesktop.Clicked += ExitToDesktop_Clicked;
+        // Close Button
+        var _buttonContinue = new Button(container, "CloseButton") { Text = Strings.EscapeMenu.Close };
```

Comment:
Don't inline this

Reply:
Also local variables (e.g. `var variableName`) should **not** have a preceding underscore, the underscore is for private fields of a type (i.e. `_buttonContinue` should be `buttonContinue`)